### PR TITLE
us3-vehicle-description

### PR DESCRIPTION
**Pull request by YannRaulet**
Display of the description of each vehicle, modification of the carousel and modification of the footer

### CODE REVIEW

**Review by erguculu — DISMISSED** [review]

File: src/Controller/ProductController.php [hunk 1]
```diff
@@ -36,12 +36,17 @@ public function chargingStations(): Response
...
-    public function vehicle(): Response
+    public function vehicle(VehicleRepository $vehicleRepository, int $id): Response
     {
-        return $this->render('front/product/vehicle.html.twig');
+        return $this->render('front/product/vehicle.html.twig', [
+            'vehicle' => $vehicleRepository->findOneBy(
+                ['id' => $id
+                ]
+            )
+        ]);
```
Comment (RomainClair) [inline src/Controller/ProductController.php]: findOneBy with an id should be replaced by find()

**Review by RomainClair — CHANGES_REQUESTED** [review]
The .github/workflows/deploy.yml and main.yml must not be removed !
This is the configuration of or CI/CD tools

File: src/Controller/ProductController.php (hunk 1 above)
Comment (YannRaulet) [reply src/Controller/ProductController.php]: I will change te findOneBy by find()

**Review by erguculu — APPROVED** [review]

File: src/Controller/ProductController.php (hunk 1 above)
Comment (YannRaulet) [reply src/Controller/ProductController.php]: I forgot that I had already tested with the find (), but it does not return to the vehicle page while with the findOneBy it works.

**Pull request closed without merging**